e2e/tests/overview-page.spec.ts

```diff
@@ -33,15 +33,17 @@ test('matches the screenshot', async ({page}) => {
 test('shows an error that their query is invalid', async ({page}) => {
   await page.goto('http://localhost:5555/?q=available_on%3Achrom');
 
-  await page.locator('.message').waitFor({state: 'visible'});
-  // The following works in --ui mode, but fails in batch mode.
-  // expect(page.locator('.message')).toContainText('Invalid query...');
+  await page.waitForSelector('.message:has-text("Invalid query...")', {
```

Comment:
The playwright docs discourage use of waitForSelector, which is why I switched to first doing `await page.locator('.message').waitFor(...)` and then checking the contents.  Maybe the updated playwright is making a difference here as well?

Reply:
Good catch. I reverted that and added an assertion to check the text too.